packages/protvista-filter/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "protvista-filter",
+  "version": "1.0.0",
```

Comment:
The version will be managed by lerna - not sure how it works for the first release of a component (haven't tried yet) but maybe take the version from `lerna.json` for now.

Reply:
Done, Thanks for pointing out.